README.md

```diff
@@ -1 +1,18 @@
-# tilt-extensions
\ No newline at end of file
+# Tilt Extensions
+This is the official Tilt Extension Repository. All extensions here have been vetted and approved by the Tilt Team.
+
+## Finding an Extension
+Here's a list of extensions along with a brief description. Try using your browsers search functionality to find an extension.
```

Comment:
We should also group them by type later on.

Reply:
Agreed.